### DISCUSSION TITLE
[ConversationHandler] Abort conversation from code

### DISCUSSION THREAD

**Discussion by steam3d** [python-telegram-bot/python-telegram-bot#3213] [Q&A]
I created bot based on [conversationbot.py](https://github.com/python-telegram-bot/python-telegram-bot/blob/master/examples/conversationbot.py)

User can cancel conversation using /cancel command 
 `fallbacks=[CommandHandler("cancel", cancel)],`
 
 How can i abort the conversation from code if error appears or some other unusual situation?

**Comment by Bibo-Joshi** [comment]
Please have a look at #2277

**Reply by steam3d** [reply]
I need to abort conversation. I do not need to catch error and make some stuff based on error.

**Reply by Bibo-Joshi** [reply]
Then please explain what you actually meany by "if error appears or some other unusual situation". See also [this page](../wiki/Ask-Right)

**Comment by steam3d** [comment]
I send  ConversationHandler.END to abort conversation. I wanted to more suitable way like  ConversationHandler.ERROR, but END also works
